datafusion/src/physical_plan/aggregates.rs

```diff
@@ -183,7 +182,7 @@ static TIMESTAMPS: &[DataType] = &[
 ];
 
 /// the signatures supported by the function `fun`.
-fn signature(fun: &AggregateFunction) -> Signature {
+pub fn signature(fun: &AggregateFunction) -> Signature {
```

Comment:
`pub(super)` or `pub(crate)` instead?

Reply:
tracked in https://github.com/apache/arrow-datafusion/issues/373